src/plugins/vis_type_timeseries/public/application/components/panel_config/timeseries.js

```diff
@@ -85,6 +86,22 @@ class TimeseriesPanelConfigUi extends Component {
         value: 'left',
       },
     ];
+    const tooltipModeOptions = [
+      {
+        label: intl.formatMessage({
+          id: 'visTypeTimeseries.timeseries.tooltipOptions.showAll',
+          defaultMessage: 'Show all values',
+        }),
+        value: 'show_all',
+      },
+      {
+        label: intl.formatMessage({
+          id: 'visTypeTimeseries.timeseries.tooltipOptions.showFocused',
+          defaultMessage: 'Show single value',
```

Comment:
Let's please rename this to `Show focused values`. Show single value does not really make sense, since we not really show only a single value, but if multiple values are close together we still show them all. I have the feeling it's rather confusing naming it "Show single value" and then actually showing multiple values in the tooltip.